CHANGELOG.md

```diff
@@ -1,6 +1,187 @@
 Changes
 =======
 
+# 5.12.0 / unreleased
+**Linux, Windows, Docker and Source Install**
+
+### Details
+
+5.12 marks a few major changes to the agent.
+
+In all agents, we are now able to release independent check packages, so that we can update checks independently and also release 3rd party packages from the community. To aid in this, we now have three repos: the [agent repo](https://github.com/datadog/dd-agent), the [core integrations repo](https://github.com/datadog/integrations-core), and the [extras integrations repo](https://github.com/datadog/integrations-extras). Core integrations are officially supported by Datadog, extras are supported by the community.
+
+The Windows agent is now built with Omnibus.
+
+The Windows agent is now always installed as a "per-machine"; previous versions installed as "per-user" by default.  Upgrades from "per-user" to "per-machine" are performed using the executable (.exe) distribution.
+
+The Windows agent is now digitally signed.
```

Comment:
Adding a comment here so that we don't forget:
We should add a link to the docs page that explains what customers upgrading from the windows agent versions < 5.12 should do, depending on the way they've installed their current Agent.
We may also want to have a dedicated `### Windows packaging changes` section here.